moped-editor/src/views/projects/projectView/ProjectComponents/EditAttributesModal.js

```diff
@@ -102,12 +102,21 @@ const EditAttributesModal = ({
       })
         .then(() => {
           onComponentSaveSuccess(updatedClickedComponentState);
-          // Zoom to the new or existing signal
-          zoomMapToFeatureCollection(
-            mapRef,
-            { type: "FeatureCollection", features: [signalFromForm] },
-            fitBoundsOptions.zoomToClickedComponent
-          );
+          const [existingLongitude, existingLatitude] =
+            featureSignalRecord.geography.coordinates[0];
+          const [newLongitude, newLatitude] =
+            signalFromForm.geometry.coordinates;
+          const hasLocationChanged =
+            existingLongitude !== newLongitude ||
+            existingLatitude !== newLatitude;
```

Comment:
Maybe this belongs in a helper?

Reply:
Do you think they will / are being used in other places? Take this with a huge grain of salt, but I'd wait until they are unless they are distracting or otherwise untidy in here. I think they look good, but you have a ton more experience in this code, to say the least, so please go with what you think is best.

Reply:
I like that take so I'm going to leave it here - thanks, Frank!